unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,117 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import time
+import pyrealsense2 as rs
+
+from rspy import test
+
+
+def get_connected_device():
+    """
+    Get first connected device
+    :return: connected device
+    """
+    dev = None
+    try:
+        context = rs.context()
+        dev = context.devices[0]
+    except Exception as ex:
+        print('Exception: Failed to find connected device. ', str(ex))
+    return dev
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :param: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def start_sensor_test(sensor, global_time_enabled: int):
+    """
+    Perform depth sensor test according given global time
```

Comment:
Not only depth right?

Reply:
The reason why I am not good with comments - a code was been changed but comment is not...
Comment changed.